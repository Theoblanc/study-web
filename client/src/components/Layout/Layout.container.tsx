import React from "react";
import LayoutPresenter from "./Layout.presenter";
import cookie from "js-cookie";
import Router from "next/router";

const LayoutContainer: React.FC = ({ children }: any) => {
  console.log("cookie", cookie.get(""));
  const token = cookie.get("accessToken");

  console.log("token", token);
  // if (token) {
  //   Router.push("/main");
  // }

  return <LayoutPresenter>{children}</LayoutPresenter>;
};

export default LayoutContainer;
