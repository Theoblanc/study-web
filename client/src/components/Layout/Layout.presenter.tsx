import React from "react";

import { Layout } from "./Layout.styled";

const LayoutPresenter = ({ children }) => {
  return <Layout>{children}</Layout>;
};

export default LayoutPresenter;
