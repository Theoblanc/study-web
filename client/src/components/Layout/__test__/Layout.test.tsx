import React from "react";
import Layout from "../Layout.container";
import renderer from "react-test-renderer";

describe("[Layout render]", () => {
  const container = renderer.create(<Layout></Layout>).toJSON();
  console.log(container);

  it("[should be render", () => {
    expect(container).toMatchSnapshot();
  });
});
