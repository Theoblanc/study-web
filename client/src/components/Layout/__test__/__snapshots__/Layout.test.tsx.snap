// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Layout render] [should be renser 1`] = `
<div
  className="sc-AxjAm StDqN"
/>
`;
